docs/system/system-metrics.md

```diff
@@ -514,6 +514,7 @@ This metric is [recommended][MetricRecommended].
 
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
+| [`system.device`](/docs/attributes-registry/system.md) | string | Unique identifier for the device responsible for managing paging operations. | `/dev/dm-0` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
It looks like duplicate of `device.id` attribute - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/device.md

Reply:
/cc @open-telemetry/semconv-mobile-approvers - is  there any reason not to reuse `device` namespace for non-mobile devices?

Reply:
It does look a lot like `device.id`. I don't see any reason why it couldn't be reused for non-mobile devices, although I just noticed that there's a note [here [1]](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/device.md) mentioning mobile specific rules around `device.id` for some reason, which I guess can be interpreted as it being a mobile-only attr. I'm up for reusing attrs as much as possible and I don't see an issue with reusing `device.id`, although we should probably update its note to avoid mentioning mobile-specific platforms (unless there is an important reason constraint it to mobile only, which I'm not aware of).

Reply:
Makes sense to me, we should avoid having duplicated attributes. I had a quick check, and it looks like many system metrics are currently using the `system.device` attribute, opened this issue to track the replacement; https://github.com/open-telemetry/semantic-conventions/issues/1432

Reply:
@open-telemetry/semconv-system-approvers wdyt?

TBH it looks really weird to me mixing these 2. How a value of `/dev/dm-0` can be represented by the same attribute that  represents the mobile device `2ab2916d-a51f-4ac8-80ee-45ac31a28092`?
If I understand it correctly the `system.device` is used for network, storage, filesystem etc devices that are part of a wider system (i.e.), a host. For example I plug an extra network interface to a physical host and I get an additional `system.device` with its own traffic metrics.

But how this can be mixed with mobile devices? 

Additionally, on entity level that's also wrong: a mobile device can plug external storage devices like `/dev/disk0` so in that case what is the `device.id`? We can find lot's of such inconsistencies specially on Android which is linux based I guess.

Reply:
> How a value of `/dev/dm-0` can be represented by the same attribute that represents the mobile device `2ab2916d-a51f-4ac8-80ee-45ac31a28092`?

Viewing it from that perspective, it seems like the term `device` might be overloaded then. The idea I had in mind as "device" was as an independent "machine", e.g. mobile phone, laptop, watch, etc, that produces telemetry. However, if we're talking about the notion of identifying mounted devices in a host machine then it does sound quite strange to mix them up.

Reply:
We have hardware attributes that would cover hardware components - 

https://github.com/open-telemetry/semantic-conventions/blob/b780f480e73ffc2d1da117689475b7b9c8af610b/model/hardware/registry.yaml#L7-L19

could we reuse them here?

Reply:
That won't work either I think. We could have cases in virtual machines where we have virtual network interfaces, disks etc. I believe we don't need to re-use anything here, `system.device` is quite accurate and represents how a system sees and handles any kind of device (systemically).

Reply:
thanks for the explanation! I'm fine with a new attribute if there is a consensus within @open-telemetry/semconv-system-approvers that's necessary,

I'd still suggest to call it `system.device.id` or `system.device.name` to be specific and have a room to add other properties.

Reply:
Hum but `system.device` is not a new attribute, this PR simply references to it: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/system.md#general-system-attributes.

I guess it makes sense to add it then first, and later as a followup, deprecate it in favor of `system.device.id|name`.

Reply:
Oh, you're right